content/references/protocol-reference/transactions/transaction-types/payment.md

```diff
@@ -33,6 +33,8 @@ Payments are also the only way to [create accounts](#creating-accounts).
 }
 ```
 
+[Query example transaction. >](websocket-api-tool.html?server=wss%3A%2F%2Fxrplcluster.com%2F&req=%7B%22id%22%3A%22example_Payment%22%2C%22command%22%3A%22tx%22%2C%22transaction%22%3A%2228E1E08E9913E1B890E4A811C4EFC614361BFFA90BA64BF980CCD429B21057B7%22%2C%22binary%22%3Afalse%7D)
```

Comment:
I'm not sure who the sender of this transaction is, but it's not great that their account name starts with "rAPER".

Let's replace that example with, I think, the actual transaction used as an example in the JSON example above (a 1 USD token payment between two of my accounts):

```suggestion
[Query example transaction. >](websocket-api-tool.html?server=wss%3A%2F%2Fxrplcluster.com%2F&req=%7B%22id%22%3A%22example_Payment%22%2C%22command%22%3A%22tx%22%2C%22transaction%22%3A%227BF105CFE4EFE78ADB63FE4E03A851440551FE189FD4B51CAAD9279C9F534F0E%22%2C%22binary%22%3Afalse%7D)
```